README.md

```diff
@@ -1,31 +1,28 @@
 # probeinterface
 
-Python package to handle probe layout, geometry and wiring to device.
-
+A Python package to handle the layout, geometry, and wiring of silicon probes for extracellular electrophysiology experiments.
 
 Documentation : https://probeinterface.readthedocs.io/
 
 
+## Goals
 
+Make a lightweight package to handle:
 
-## Goal
-
-Make a ligthweigted package usefull that handle:
-
-  * probe geometry (2d, 3d  layout)
+  * probe site geometry (both 2D and 3D layouts)
```

Comment:
Is it possible to use 'contact' instead of 'site'. To make the doc more consistent.

Reply:
Ok, fixed!